### ISSUE TITLE
Improve change network feature

### ISSUE THREAD

**Issue by ilanolkies** [rsksmart/rLogin#277]
TLDR; this is one of the features the developer can use _after_ user has logged in. The idea is that multi-chain apps have a simplified manner to let the user switch networks.

Now it is just a _refactored_ feature from _wrong network_. We can improve this a lot, we need to discuss how.

This is also related to https://github.com/rsksmart/rLogin/issues/275 and https://github.com/rsksmart/rLogin/issues/268
---

This new language makes more sense but the functionality isn't implemented. So if the user is using Metamask then they will see that they can automatically connect to RSK Mainnet. But it also shows that the app can change to Ethereum but there is no way for the user to do so. They have to do it from their Provider.

If we want to include this functionality, then we should also implement the Metamask specific`wallet_addEthereumChain` method for the other networks. In that case, in #274 we should combine the information from [the adapters file](https://github.com/rsksmart/rLogin/pull/274/files#diff-5434d7621041e2bbe017e128e361d977a1e34958ea9a3ed45a2ea3e4450b9820R15-R58) with the [networks variable](https://github.com/rsksmart/rLogin/blob/develop/src/ux/wrongNetwork/changeNetwork.ts#L14-L43). There is a lot of duplication there and by single sourcing it developers can add additional networks quicker. 

That `networks` variable also includes a link to the blockexplorer which could be the default and then overwritten by the developer's option?

_Originally posted by @jessgusclark in https://github.com/rsksmart/rLogin/pull/273#discussion_r880322032_

**Comment by ilanolkies** [comment]
Also related to https://github.com/rsksmart/rLogin/issues/181
